src/ui/components/TestSuite/views/TestItem/TestSteps/NavigationTestStepRow.tsx

```diff
@@ -9,10 +9,15 @@ export default memo(function NavigationTestStepRow({
 }: {
   navigationTestStep: NavigationTestStep;
 }) {
+  let url = navigationTestStep.data.url || "";
+  let displayUrl = url.length > 500 ? url.slice(0, 500) + "..." : url;
```

Comment:
I'm confused. Wouldn't this do 500 when the title says 250?

Reply:
Whoops, thanks. I committed two things:

1. Limit to 500, then change to 250
2. Check for null

When I did step 2, I mistakenly copied over the 500. Fixed now!